core/src/com/po/fuck/Constants.java

```diff
@@ -20,4 +20,10 @@ public final class Constants {
     public static float HEALTHBAR_OFFSET = 5;
     public static final int ENEMY_TEAM_TAG = 0;
     public static final int PLAYER_TEAM_TAG = 1;
+
+    public static final float spriteCoinPositionX = 80;
```

Comment:
Rename them as other constants(SCREAMING_SNAKE_CASE), and a proper tag `/* coins ui */`